amf-shapes/shared/src/main/scala/amf/shapes/client/platform/model/domain/jsonldinstance/JsonLDArray.scala

```diff
@@ -0,0 +1,19 @@
+package amf.shapes.client.platform.model.domain.jsonldinstance
+
+import amf.core.client.scala.model.domain.AmfArray
+import amf.shapes.client.scala.model.domain.jsonldinstance.{JsonLDArray => InternalJsonLDArray}
+import amf.shapes.internal.convert.ShapeClientConverters.ClientList
+import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
+
+@JSExportAll
+class JsonLDArray extends AmfArray(Nil) with JsonLDElement {
```

Comment:
You should have the _internal as the other interfaces. If not it will be difficult to implement the converter.